tests/test_fdb.py

```diff
@@ -122,10 +86,114 @@ def test_FDBAddedAfterMemberCreated(dvs):
                     [("mac", "52-54-00-25-06-E9"), ("vlan", "2")],
                     [("SAI_FDB_ENTRY_ATTR_TYPE", "SAI_FDB_ENTRY_TYPE_DYNAMIC"),
                      ("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID", iface_2_bridge_port_id["Ethernet0"]),
                      ('SAI_FDB_ENTRY_ATTR_PACKET_ACTION', 'SAI_PACKET_ACTION_FORWARD')]
     )
     assert ok, str(extra)
+
+
+def test_fdb_notifications(dvs):
+    dvs.setup_db(dvs)
+
+    # create vlan; create vlan member
+    dvs.create_vlan(dvs, "6")
+    dvs.create_vlan_member("6", "Ethernet64")
+    dvs.create_vlan_member("6", "Ethernet68")
+
+    # bring up vlan and member
+    dvs.set_interface_status("Vlan6", "up")
+    dvs.add_ip_address("Vlan6", "6.6.6.1/24")
+    dvs.set_interface_status("Ethernet64", "up")
+    dvs.set_interface_status("Ethernet68", "up")
+    dvs.servers[16].runcmd("ifconfig eth0 6.6.6.6/24 up")
+    dvs.servers[16].runcmd("ip route add default via 6.6.6.1")
+    dvs.servers[17].runcmd("ifconfig eth0 6.6.6.7/24 up")
+    dvs.servers[17].runcmd("ip route add default via 6.6.6.1")
+
+    # get neighbor and arp entry
+    rc = dvs.servers[16].runcmd("ping -c 1 6.6.6.7")
+    assert rc == 0
+
+    # Get mapping between interface name and its bridge port_id
+    iface_2_bridge_port_id = get_map_iface_bridge_port_id(dvs.adb, dvs)
+
+    # check that the FDB entries were inserted into ASIC DB
+    ok, extra = dvs.is_fdb_entry_exists(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_FDB_ENTRY",
+                    [],
+                    [("SAI_FDB_ENTRY_ATTR_TYPE", "SAI_FDB_ENTRY_TYPE_DYNAMIC"),
+                     ("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID", iface_2_bridge_port_id["Ethernet64"]),
+                    ]
+    )
+    assert ok, str(extra)
+    ok, extra = dvs.is_fdb_entry_exists(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_FDB_ENTRY",
+                    [],
+                    [("SAI_FDB_ENTRY_ATTR_TYPE", "SAI_FDB_ENTRY_TYPE_DYNAMIC"),
+                     ("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID", iface_2_bridge_port_id["Ethernet68"]),
+                    ]
+    )
+    assert ok, str(extra)
+
+    # check that the FDB entries were inserted into State DB
+    ok, extra = dvs.is_table_entry_exists(dvs.sdb, "FDB_TABLE",
+                    "Vlan6:.*",
+                    [("port", "Ethernet64"),
+                     ("type", "dynamic"),
+                    ]
+    )
+    assert ok, str(extra)
+    ok, extra = dvs.is_table_entry_exists(dvs.sdb, "FDB_TABLE",
+                    "Vlan6:*",
+                    [("port", "Ethernet68"),
+                     ("type", "dynamic"),
+                    ]
+    )
+    assert ok, str(extra)
+
+    # enable warm restart
+    # TODO: use cfg command to config it
+    create_entry_tbl(
+        dvs.cdb,
+        swsscommon.CFG_WARM_RESTART_TABLE_NAME, "swss",
+        [
+            ("enable", "true"),
+        ]
+    )
+
+    try:
+        # restart orchagent
+        dvs.runcmd(['sh', '-c', 'supervisorctl restart orchagent'])
```

Comment:
Would it be better to do swss start/stop?   If it is for warm restart testing,  test_warm_reboot.py might be a good place for all related test cases.    
```
# start processes in SWSS
def start_swss(dvs):
    dvs.runcmd(['sh', '-c', 'supervisorctl start orchagent; supervisorctl start portsyncd; supervisorctl start intfsyncd; \
        supervisorctl start neighsyncd; supervisorctl start intfmgrd; supervisorctl start vlanmgrd; \
        supervisorctl start buffermgrd; supervisorctl start arp_update'])

# stop processes in SWSS
def stop_swss(dvs):
    dvs.runcmd(['sh', '-c', 'supervisorctl stop orchagent; supervisorctl stop portsyncd; supervisorctl stop intfsyncd; \
        supervisorctl stop neighsyncd;  supervisorctl stop intfmgrd; supervisorctl stop vlanmgrd; \
        supervisorctl stop buffermgrd; supervisorctl stop arp_update'])

```